tests/unit/spec/firefox/whatsnew/whatsnew-62.js

```diff
@@ -0,0 +1,82 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect */
+
+describe('whatsnew-62.js', function () {
+    'use strict';
+
+    describe('init', function () {
+
+        it('should call showPocket if user is not logged in to FxA and lang is en', function () {
+            spyOn(Mozilla.WNP62, 'showPocket');
+
+            Mozilla.WNP62.init({
+                // TODO lang: en
```

Comment:
If it's one of the English locales it fires showPocket, else it fires showFxa. See https://github.com/mozilla/bedrock/pull/6090/files#diff-0595be2fdb4544a0494cd69a9451ec4dR138 for reference. I'm not sure how we can test for this (I have no idea how to write these tests, I just copied this from whatsnew-61).

Reply:
First, these tests aren't being run just yet. You need to add the following files to `tests/unit/karma.conf.js`

- 'media/js/firefox/whatsnew/whatsnew-62.js',
- 'tests/unit/spec/firefox/whatsnew/whatsnew-62.js',

Reply:
```javascript
        afterEach(function() {
            Mozilla.WNP62.locale = null;
        });

        it('should call showPocket if user is not logged in to FxA and lang is en', function () {
            Mozilla.WNP62.locale = 'en';
            spyOn(Mozilla.WNP62, 'showPocket');

            Mozilla.WNP62.init({
                setup: false
            });

            expect(Mozilla.WNP62.showPocket).toHaveBeenCalled();
        });

        it('should call showFxa if user is not logged in to FxA and lang is *not* en', function () {
            Mozilla.WNP62.locale = 'fr';
            spyOn(Mozilla.WNP62, 'showFxa');

            Mozilla.WNP62.init({
                setup: false
            });

            expect(Mozilla.WNP62.showFxa).toHaveBeenCalled();
        });
```